pkg/serverless/trace/inferredspan/span_creation.go

```diff
@@ -0,0 +1,103 @@
+package inferredspan
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+func CreateInferredSpanFromAPIGatewayEvent(eventSource string, attributes EventKeys, inferredSpan InferredSpan) {
```

Comment:
It looks like this function is specifically for REST APIs -- I would rename the function to reflect that